File: JavaNgArch/src/javangarch/domain/bdinterface/IAdminBD.java



package javangarch.domain.bdinterface;

import javangarch.domain.domainmodel.gn.Corp;
import javangarch.domain.domainmodel.gn.User;
import projectbase.domain.IBusinessDelegate;

    public interface IAdminBD extends IBusinessDelegate 
    {
        User GetLoginUser(String corpCode, String userCode, String password);
        void RefreshUser(User user);
        void RefreshCorp(Corp corp);
    }

File: NetNgArch/src/NetNgArch.Domain/BDInterface/IAdminBD.cs

using System.Collections.Generic;
using NetNgArch.Domain.DomainModel.GN;
using ProjectBase.Domain;

namespace NetNgArch.Domain.BDInterface
{
    public interface IAdminBD :IBusinessDelegate 
    {
        User GetLoginUser(string userCode,string password);
        void RefreshUser(User user);
        void RefreshCorp(Corp corp);
    }
}
